app/models/user.rb

```diff
@@ -53,15 +53,20 @@ class User < ApplicationRecord # rubocop:disable Metrics/ClassLength
                           foreign_key: "banned_by_id",
                           dependent:   :nullify
 
-  SCREEN_NAME_REGEX = /\A[a-zA-Z0-9_]{1,16}\z/
+  SCREEN_NAME_REGEX = /\A[a-zA-Z0-9_]+\z/
   WEBSITE_REGEX = /https?:\/\/([A-Za-z.-]+)\/?(?:.*)/i
 
   before_validation do
     screen_name.strip!
   end
 
   validates :email, fake_email: true, typoed_email: true
-  validates :screen_name, presence: true, format: { with: SCREEN_NAME_REGEX }, uniqueness: { case_sensitive: false }, screen_name: true
+  validates :screen_name,
+            presence:    true,
+            format:      { with: SCREEN_NAME_REGEX },
```

Comment:
would add a message to it too so it's not just "invalid"
```suggestion
            format:      { with: SCREEN_NAME_REGEX, message: "contains invalid characters" },
```

Reply:
Hmm, fair point, but in combination with the format hint right below it is kinda explaining why it is "invalid", isn't it?

Reply:
i'd rather have error messages be more explicit, even with the help text below.  e.g. `ü` would count as an alphanumeric character ;-)